utils/converter/src/main/java/io/apicurio/registry/utils/converter/AbstractConverter.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.utils.converter;
+
+import io.apicurio.registry.utils.IoUtil;
+import io.apicurio.registry.utils.serde.AbstractKafkaSerDe;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serializer;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaAndValue;
+import org.apache.kafka.connect.storage.Converter;
+
+import java.util.Map;
+
+/**
+ * Very simplistic converter -- no Schema handling atm.
+ *
+ * @author Ales Justin
+ */
+public class AbstractConverter<T> extends AbstractKafkaSerDe implements Converter, AutoCloseable {
```

Comment:
Yeah ... I thought about it ... and left it as such, since it's usable anyway.
I'll rename it to SchemalessConverter, since it doesn't handle Schemas as needed/required, but it's still good enough, since it does do serde stuff.